filebeat/input/log/input.go

```diff
@@ -11,8 +11,8 @@ import (
 
 	"github.com/elastic/beats/filebeat/channel"
 	"github.com/elastic/beats/filebeat/harvester"
+	input "github.com/elastic/beats/filebeat/input"
```

Comment:
Why do you prefix it here with `input`? It should already be input.

Reply:
let me check that, i believe my editor did this change.